jsk_rviz_plugins/src/overlay_camera_display.cpp

```diff
@@ -398,7 +398,7 @@ void OverlayCameraDisplay::update( float wall_dt, float ros_dt )
   if (!overlay_) {
     static int count = 0;
     rviz::UniformStringStream ss;
-    ss << "OverlayImageDisplayObject" << count++;
+    ss << "OverlayCameraImageDisplayObject" << count++;
```

Comment:
Thank you! this is typo, I think.

can you change like this?

```suggestion
    ss << "OverlayCameraDisplayObject" << count++;
```

as https://github.com/jsk-ros-pkg/jsk_visualization/blob/7d31711959cfe88eaef8fc8a426accaf6dd02621/jsk_rviz_plugins/src/overlay_camera_display.cpp#L182

Reply:
Changed and tested.

Reply:
Unfortunately, with this change, it no longer segfaults, but it also doesn't draw any Rviz items except for the image itself. I'll revert this change.